test/WebJobs.Script.Tests.Integration/WebJobs.Script.Tests.Integration.csproj

```diff
@@ -42,12 +42,10 @@
     </PackageReference>
     <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="8.0.1" />
     <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="2.11.2" />
-    <PackageReference Include="Microsoft.Azure.EventHubs" Version="2.1.0" />
```

Comment:
We have one _skipped_ EndToEnd integration test `SamplesEndToEndTests_Node_EventHubTrigger` making use of this deprecated EventHub package. We should either try to fix it or remove it completely.